cli/src/pcluster/config/cluster_config.py

```diff
@@ -1200,6 +1210,25 @@ def mount_dir_list(self):
 
         return mount_dir_list
 
+    @property
+    def local_mount_dir_set(self):
+        """Retrieve the list of local mount dirs for the shared storage and head node ephemeral volume."""
+        mount_dir_set = {
+            self.head_node.local_storage.ephemeral_volume.mount_dir
+            if self.head_node.local_storage.ephemeral_volume
+            else "/scratch"
```

Comment:
let's add this into a constant, I saw this string in too many places

Reply:
Done